queue-client/healthcheck-cli/src/main/java/org/verapdf/webapp/queueclienthealthcheckcli/listener/QueueListenerImpl.java

```diff
@@ -16,13 +16,14 @@
 	private Set<String> successfullyReceivedMessages = new HashSet <>();
 
 	@Override
-	public void process(String message) {
+	public String process(String message) {
 		try {
 			Thread.sleep(1000);
 		} catch (InterruptedException e) {
 			LOGGER.info("Caught exception while waiting for processing jobId by worker.");
 		}
 		successfullyReceivedMessages.add(message);
+		return null;
```

Comment:
it would be grate to respond with the initial data to be sure that the response works correctly